.env.local

```diff
@@ -0,0 +1 @@
+REACT_APP_INFURA_PROJECT_ID=57047f35d665403dbad08deee19ceee1
```

Comment:
Might be a better idea to inject this into the environment when we run it rather than committing it.

Reply:
Yeah, let me try to make that better.  I was following the README: 

https://github.com/WalletConnect/walletconnect-test-wallet/blob/master/README.md